migration/validator-activation.md

```diff
@@ -0,0 +1,76 @@
+
+- **IPFS**: Download the full PoW chain snapshot [here](https://ipfs.nimiq.io/ipfs/QmRKvFVpTdXagvgZG5cF9qdz13x9DkZhUvwXAS5YMaqTfu?filename=pow-main-full-consensus.zip)
+- **Torrent**: Alternatively, download the snapshot via Torrent [here](https://repo.nimiq.com/torrents/nimiq-pow-main-full-consensus.torrent)
+
+After downloading the snapshot, the database file must be placed in the directory from where you are running your PoW node. Once in place, follow steps [3](#step-3-prepare-the-pow-client-for-the-transition) and [4](#step-4-run-the-activation-tool). After your node reaches consensus, continue with the Activation Tool to complete the process.
+
+### **Step 3: Prepare the PoW Client for the Transition**
+Ensure the **PoW client is running with RPC enabled** on a distinct port from the PoS client. Configure the PoW client’s RPC settings in its configuration file by adding your validator address and key pair. Refer to [**this example**](https://github.com/nimiq/core-js/blob/master/clients/nodejs/sample.conf#L163).
+
+Run the following command to start the PoW client:
+
+```bash
+    node clients/nodejs/index.js --dumb --network=main --rpc=8648 --wallet-seed=private_keypublic_key
+```
+
+The `private_keypublic_key` is the concatenation of your private key and public key in that order
+
+### **Step 4: Run the Activation Tool**
+Once your PoW client is synced and in consensus, you are ready to run the Validator Activation Tool. This tool establishes a connection with the PoW chain via RPC and sets your validator ready for the PoS transition.
+
+1. Compile the PoS Client
+Ensure your PoS client is [properly configured](#step-1-add-validator-data-into-the-configuration-file) in the `client.toml` file, and your validator data is correctly set, and compile the PoS client:
+    ```
+    cargo build --release
+    ```
+2. **Run the Activation Tool** using the following command in the PoS client directory:
+
+    ```
+    cargo run --release --bin nimiq-pow-migration --url "pow-rpc" --config client.toml
+    ```
+
+    Where `pow-rpc` is the **PoW client RPC url**. For example: `http://127.0.0.1:8648`
+
+The tool will monitor validator readiness by tracking readiness transactions sent by all the registered validators within 24 hour activation windows. Once 80% of the total stake has signaled readiness, the transition block will be generated. Read more about the activation process [here](migration-technical-details#readiness-and-activation-tool).
+
+### PoS Activation
+
+On **November 19**, the network will initiate the transition from PoW to PoS. Once 80% of the total stake signals their readiness, the activation process will begin. The transition block will be generated, and validators will officially start securing the PoS network. Ensure your validator is ready and synced before this date to avoid delays.
```

Comment:
Same as above. On november 19th the activation phase starts. Once 80% of the total state signal their rediness the PoS chain starts with the candidate block as the genesis block.